packages/validator/src/services/indices.ts

```diff
@@ -110,6 +116,48 @@ export class IndicesService {
         newIndices.push(validatorState.index);
       }
     }
+
+    logValidatorStatuses(allValidators, this.logger);
+
     return newIndices;
   }
 }
+
+function logValidatorStatuses(allValidators: Map<ValidatorStatus, ValidatorResponse[]>, logger: Logger): void {
+  allValidators.get("active")?.forEach((validatorState: ValidatorResponse) => {
+    logger.info("Validator activated", {
+      pubkey: toHexString(validatorState.validator.pubkey),
+      index: validatorState.index,
+    });
+  });
+
+  [...(allValidators.get("pending_initialized") ?? []), ...(allValidators.get("pending_queued") ?? [])]?.forEach(
```

Comment:
IMO this line is a bit hard to read, the logic is also repeated in multiple places

maybe can do something like this:

add a function/method (maybe should use class method and store allValidators as class property? and then access via `this.allValidators` instead of passing it into multiple functions)

```ts
function getValidatorStatesByStatuses(statuses: ValidatorStatus[], allValidators: Map): ValidatorResponse[] {
   return statuses.filter((status)=> allValidators.has(status)).map((status) => allValidators.get(status)
}

```

and then here we can just do

```suggestion
  getValidatorStatesByStatuses(["pending_initialized", "pending_queued"]).forEach(
```